manifests/modelserving/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+commonLabels:
+  app: odh-dashboard
+  app.kubernetes.io/part-of: odh-dashboard
+resources:
+  -  servingruntimes-template.yaml
```

Comment:
add 
```
images:
- name: openvino-model-server
  newName: quay.io/modh/openvino-model-server
  digest: sha256:c89f76386bc8b59f0748cf173868e5beef21ac7d2f78dada69089c4d37c44116
```
to avoid directly referencing image values in manifests.